project-document/docs/domain-model/group-context/group.md

```diff
@@ -0,0 +1,24 @@
+# Group
+
+## Group ID (Identifier)
+
+The Group ID is generated by the internal system.
+The ID style is to be determine
+
+
+## Group (Aggregate Root)
+
+The Group is an group including userIds and containerIds.
+
+### Group Name
+
+The group can set a groupName as they like. It must be less than equal to 255 letters and greater than equal to 1.
+This name can change anytime. it has default name when the group created without specifying the name.
+
+## User IDs
+
+the group is composed of users. The group class has user Id list. The Group **cannot exist** without any user.
```

Comment:
This should be a capital letter. And "Id" should be ID because this is just a document not a code.